units/commit_queue.go

```diff
@@ -316,6 +317,7 @@ func (j *commitQueueJob) processGitHubPRItem(ctx context.Context, cq *commitqueu
 		j.logError(err, "can't finalize patch", nextItem)
 		j.dequeue(cq, nextItem)
 		j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't finalize patch", ""))
+		return
 	}
 
 	err = subscribeGitHubPRs(pr, modulePRs, projectRef, v.Id)
```

Comment:
should this also return?

Reply:
Ah I was thinking this wasn't a fatal error but I guess we do dequeue so yes.